src/StreamFactoryInterface.php

```diff
@@ -9,15 +9,11 @@
     /**
      * Create a new stream from a resource or a string.
      *
-     * If the argument is a resource, it MUST be readable and SHOULD be seekable. It MAY be writable.
-     * If the argument is a string, a temporary resource will be created that is writable and seekable.
-     * 
-     * If the arugment is a string it will be interpided as the content of the stream. File names or 
-     * file paths are not supported. 
+     * String content will be used as the body of the stream. Files paths are not supported.
```

Comment:
> If we can reduce interoperability concerns to a single, consistent interface, and this enables someone to write a utility library that provides the same functionality and utility for all implementations of that interface, isn't that the ideal?

> In my opinion, if we design the interfaces not based on what's minimally required to level the differences between libraries, but instead design it to meet our own personal ideals or preferences about what such a library should look like, we're unnecessarily burdening developers with extra work to satisfy our needs.

> I think it's really important to remember that we're not designing a library here - we're designing the layer that is supposed to help libraries be interchangible.

This is where I think you are wrong. 

We are designing an interface that library authors should rely on. If someone (or we) creates an utility library, I would consider that as a failure for PSR-17. 

I strongly believe we should consider usability when we are designing a PSR.